GETTING_STARTED.md

```diff
@@ -0,0 +1,415 @@
+- Eunomia watching for changes in a remote repository and acting upon them.
+
+## Prerequisites
+
+In order to build Eunomia, you'll need the following:
+
+- [Git](https://git-scm.com/downloads)
+- [Go 1.12+](https://golang.org/dl/)
+- [Docker](https://docs.docker.com/install/)
+- [Operator SDK v0.8.1](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md)
+- Access to a Kubernetes cluster
+  - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+  - [Minikube](https://kubernetes.io/docs/setup/minikube/) (for local development and testing)
+
+You will also have to [set up the GOPATH](https://github.com/golang/go/wiki/SettingGOPATH) env variable (the instructions are for Go 1.13 but it's the same for Go 1.12).
+
+### Installing on a Mac
+
+All the components (except operator-sdk v0.8.1) can be easily installed via [Homebrew](https://brew.sh/) on a Mac:
+
+```shell
+brew install git
+brew install go@1.12
+brew install docker
+brew install minikube
+brew install kubectl
+```
+
+Unfortunately, operator-sdk in version 0.8.1 isn't available via brew so you have to install it from GitHub release as described [on this page](https://github.com/operator-framework/operator-sdk/blob/v0.8.1/doc/user/install-operator-sdk.md).
+
+### Checkout your fork
+
+You wil need your own Eunomia fork later in the second example. When pushing changes to your fork, you will observe Eunomia make adjustments based on your changed GitOpsConfig.
+
+Check out [the Eunomia repository](https://github.com/KohlsTechnology/eunomia).
+
+1. Create a [fork](https://help.github.com/en/articles/fork-a-repo) of it
+2. Create the directories and clone your fork:
+
+```
+mkdir -p ${GOPATH}/src/github.com/KohlsTechnology
+cd ${GOPATH}/src/github.com/KohlsTechnology
+git clone https://github.com/<YOUR FORK>/eunomia.git
+cd eunomia
+```
+
+## Deploy Eunomia to a Minikube cluster
+
+### Start a local Minikube cluster
+
+Minikube is a minimal Kubernetes cluster run in a virtual machine in Virtualbox.
+
+```shell
+minikube start
+```
+
+From now on your local Kubernetes client (kubectl) is configured to use your just started Minikube cluster.
+
+### Deploy images to Minikube Docker registry
+
+The Eunomia operator gets packaged as a container image - we need to build it (with some other template processor images which [are described here](./README.md#Template-Engine)) and send to Minikube Docker registry to be easily accessible from the cluster.
+
+To achieve this run the helper script (assuming you are in your local Eunomia repository dir):
+
+```shell
+./scripts/deploy-to-local.sh minikube
+```
+
+### Deploy Eunomia to your Minikube cluster
+
+It'll be done with the help of [Helm](https://helm.sh/). If you use a Mac, you can easily install it via Homebrew.
+
+```shell
+helm template deploy/helm/eunomia-operator/ \
+  --set eunomia.operator.image.tag=dev \
+  --set eunomia.operator.image.pullPolicy=Never | kubectl apply -f -
```

Comment:
Similar as above: don't we want users to stay with latest images, instead of dev? During our development, we specifically override tag=dev and pullPolicy=Never, so that we can leave latest and Always as defaults for end users. If we want to teach end users to also use dev and Never, there would be not much sense to keep latest and Always in the repo; but it's not yet clear to me why we would now want to teach users to use values other than latest and Always?